fecgov/openFEC
# Update GPO statute link format

## Summary (required)

- Partial resolution for #3942 

Update GPO statute link format

See https://www.govinfo.gov/link-docs/

Before (broken): http://api.fdsys.gov/link?collection=uscode&title=26&year=mostrecent&section=9003
After: https://www.govinfo.gov/link/uscode/26/9003

_Include a summary of proposed changes._

## How to test the changes locally

- Check out this branch
- Get elasticsearch running locally
- Run `./manage.py index_statutes`
- Check out http://localhost:5000/v1/legal/search?hits_returned=20&type=statutes&from_hit=0&q=contribution
- Test URL links 


## Impacted areas of the application
List general components of the application that this PR will affect:

-  Statutes search: https://www.fec.gov/data/legal/search/statutes/?search=contribution
- Statute links in AOs and MURs: https://www.fec.gov/data/legal/advisory-opinions/2019-12/ and https://www.fec.gov/data/legal/matter-under-review/7433/
